etcd/README.md

```diff
@@ -0,0 +1,17 @@
+# Finagle Etcd client #
+
+## `io.buoyant.etcd` ##
+
+The `io.buoyant.etcd` package contains a generically useful Etcd
+client built on finagle-http. The client supports a variety of
+asynchronous `key` operations on etcd.  It goes beyond the basic etcd
+operations to provide a `watch` utility that models a key's (or
+tree's) state as an `com.twitter.util.Activity`.
+
+### TODO ###
+
+- `compareAndDelete` isn't yet implemented.  minor oversight.
+- Use residual paths on the client to instrument chrooted key
+operations
```

Comment:
clients can be constructed with names that have residual paths.  I.e. i can create a client with `/$/inet/127.1/4001` or `/$/inet/127.1/4001/some/prefix`.  These both bind clients on 127.1:4001.  The latter is bound with a residual path, `/some/prefix`.  It would be cool to use this as a prefix for all etcd operations.  I don't have a concrete use case for this right now, though.